### PR TITLE
#feat - video-listing page added.

### PR DESCRIPTION
**Pull request by surajmore98**
added:

- video listing page
- routing

### CODE REVIEW

File: src/backend/controllers/VideoController.js [hunk 1]
```diff
@@ -51,3 +51,21 @@ export const getVideoHandler = function (schema, request) {
     );
   }
 };
+
+export const getVideoByTypeHandler = function (schema, request) {
```
Comment (vinc3nati) [inline src/backend/controllers/VideoController.js:55]: getting video by type can be done in frontend too, but for large app it's a great plus point if we could do so with backend, nice work

File: src/components/video-list/video.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    },
+    {
+        headers: {
+            authorization: token
+        }
+    });
+}
+
+// remove from liked videolist
+const removeFromWatchLaterVideos = async (videoId, token) => {
```
Comment (vinc3nati) [inline src/components/video-list/video.js:44]: it's a good practice to store all your services in separate file so it becomes more legible

**Review by vinc3nati — APPROVED** [review]

**Review by lomesshh — APPROVED** [review]
clean code
The video listing page looking good
suggestion:
browser router can be added to index file instead of app.js

looks good to merge

File: src/components/video-list/video.js (hunk 2 above)
Comment (surajmore98) [reply src/components/video-list/video.js:44]: Thanks for suggestion, I will change it in next PR which is related to watchlater

**Pull request merged**